apps/studio/src/pages/sites/[siteId]/settings/notification.tsx

```diff
@@ -39,6 +40,8 @@ const NotificationSettingsPage: NextPageWithLayout = () => {
     siteId,
   })
 
+  const [, setIsDismissed] = useSessionStorage("notification-dismissed", false)
```

Comment:
should we expoprt the key `notification-dismissed` or the session storage out from components?
```ts
  const [isDismissed, setIsDismissed] = useSessionStorage(
    "notification-dismissed",
    false,
  )
```

might be more readable + more maintainable (single source of truth)

Reply:
actually you're right, i think i'll export a `useIsNotificationDismissed` hook ba